### ISSUE TITLE
Throw informative error when read_json_param_objects points to nonexistent json file

### ISSUE THREAD

**Issue by MaxGhenis** [PSLmodels/Tax-Calculator#2518]
I was helping another user who was confused by the error message shown in https://github.com/PSLmodels/Tax-Calculator/issues/2455 when `read_json_param_objects` points to a nonexistent json file:
```
ValueError: Text below contains invalid JSON:
Expecting value: line 1 column 1 (char 0)
Above location of the first error may be approximate.
The invalid JSON text is between the lines:
XXXX----.----1----.----2----.----3----.----4----.----5----.----6----.----7
0001_static/reformA.json
XXXX----.----1----.----2----.----3----.----4----.----5----.----6----.----7
```
It'd be good to check if the file exists and tell users when it doesn't.

**Comment by MattHJensen** [comment]
Resolved by #2524

**Comment by MattHJensen** [comment]
@MaxGhenis, thanks a lot for opening the issue as well as helping the other tax calculator user!
